### ISSUE TITLE
Data augmentation option

### ISSUE THREAD

**Issue by fako1024** [els0r/goProbe#287]
In the past, several suggestions have been made by various sources how the available data could be augmented to improve the value `goProbe` provides even more.

**Important:** Any modification must _NOT_ affect performance in any significant way to keep up with the high standards established by us.

Ideas:

- [ ] Store the first (/last) exact timestamp a flow was seen (maybe in an adjacent data structure so that it doesn't affect the main table)
- [ ] Track fragmented packets as parsing issues (similar to the existing one for truncated packets) to help spot issues simply by looking at the interface processing stats

**Comment by els0r** [comment]
Fully onboard, as long as we stay backwards compatible.

**Comment by fako1024** [comment]
> Fully onboard, as long as we stay backwards compatible.

Absolutely, was so obvious that I forgot to add it to the "Important" area above. Amended... :stuck_out_tongue:
